apps/aecore/test/aecore_txs_pool_test.exs

```diff
@@ -67,11 +68,15 @@ defmodule AecoreTxsPoolTest do
   end
 
   test "add negative transaction fail", wallet do
-    nonce = Map.get(Chain.chain_state().accounts, wallet.a_pub_key, %{nonce: 0}).nonce + 1
+    nonce = AccountHandler.nonce(get_accounts_chainstate(), wallet.a_pub_key) + 1
     payload = %{to_acc: wallet.b_pub_key, value: -5}
     tx1 = DataTx.init(SpendTx, payload, wallet.a_pub_key, 0, nonce)
 
     {:ok, signed_tx} = SignedTx.sign_tx(tx1, wallet.priv_key)
     assert :error = Pool.add_transaction(signed_tx)
   end
+
+  defp get_accounts_chainstate() do
```

Comment:
maybe you can add some testutils to import this functions, do not duplicate code